src/Microsoft.Health.Blob/Registration/BlobClientRegistrationExtensions.cs

```diff
@@ -131,9 +131,11 @@ public static IServiceCollection AddBlobServiceClient(this IServiceCollection se
                     IAzureClientBuilder<BlobServiceClient, BlobClientOptions> clientBuilder;
                     if (string.Equals(options.Credential, "managedidentity", StringComparison.OrdinalIgnoreCase))
```

Comment:
If this is all configuration based, perhaps this can be simplified to always specify the `WithCredential` since we're using the `DefaultCredentials`?

Reply:
There's code above to set `options.ConnectionString = BlobLocalEmulator.ConnectionString;` for testing purposes, and that won't be able to transform into a Uri below. In theory we could get rid of the Uri transformation and just always pass the connectionstring into `AddBlobServiceClient` as a string, but I'm not sure if there are any other implications